kubernetes_asyncio/client/api_client.py

```diff
@@ -188,17 +188,15 @@ async def __call_api(
             e.body = e.body.decode('utf-8') if six.PY3 else e.body
             raise e
 
-        content_type = response_data.getheader('content-type')
```

Comment:
response_data can be a raw response (aiohttp ClientResponse) if _preload_content is false. The raw response doesn't have methods like getheader, getheaders. This assign is not necessary here and can be done later when preloaded content is processing.